newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationList.js

```diff
@@ -439,36 +442,44 @@ const AnimationList = React.forwardRef<
     );
 
     const importImages = React.useCallback(
-      async () => {
+      async (resourceSource: ResourceSource) => {
         const resources = await resourceManagementProps.onChooseResource({
-          initialSourceName: resourceSources[0].name,
+          initialSourceName: resourceSource.name,
           multiSelection: true,
           resourceKind: 'image',
         });
         if (resources.length === 0) {
           return;
         }
-        resources.forEach(resource => {
-          applyResourceDefaults(project, resource);
-          project.getResourcesManager().addResource(resource);
-        });
+        if (resourceSource.shouldCreateResource) {
+          resources.forEach(resource => {
+            applyResourceDefaults(project, resource);
+            project.getResourcesManager().addResource(resource);
+          });
 
-        addAnimations(groupResourcesByAnimations(resources));
+          const resourcesByAnimation = resourceSource.shouldGuessAnimationsFromName
+            ? groupResourcesByAnimations(resources)
+            : new Map<string, Array<gdResource>>();
+          addAnimations(resourcesByAnimation);
```

Comment:
Is it useful to do if `shouldGuessAnimationsFromName` is false?

Reply:
Yes because this is the screen appearing when you don't have any animations, so we need to create one empty with the sprites selected

Reply:
I've updated this, this was wrong indeed